infrastructure/code/sast_enabled.go

```diff
@@ -32,18 +32,13 @@ const codeDisabledInOrganisationMessageText = "It looks like your organization h
 
 const enableSnykCodeMessageActionItemTitle snyk.MessageAction = "Enable Snyk Code"
 const closeMessageActionItemTitle snyk.MessageAction = "Close"
-const localCodeEngineWarning = "Snyk Code is configured to use a Local Code Engine instance. This setup is not yet supported."
 
-func (sc *Scanner) isSastEnabled() bool {
-	sastResponse, err := sc.SnykApiClient.SastSettings()
+func (sc *Scanner) isSastEnabled(sastResponse snyk_api.SastResponse) bool {
 	method := "isSastEnabled"
-	if err != nil {
-		log.Error().Err(err).Str("method", method).Msg("couldn't get sast enablement")
-		sc.errorReporter.CaptureError(err)
-		return false
-	}
 
 	if !sastResponse.SastEnabled {
+		// self-note: if LCE is enabled, this condition is satisfied. Why?
```

Comment:
What is this comment about? Don't understand it.

Reply:
Outdated. Removed